wordpress-mobile/WordPress-iOS
# Update Helpshift SDK to v6.0.0

**Fixes** #7694 

**To test:**

Make sure all the tags and custom fields are populated when contacting support and that the FAQs work as expected.
Helpshift ticket creation can be triggered at least from these three places:
- Delete Site
- Me->Help & Support
- Aztec bug reporting

For easier testing I suggest creating one ticket with the current version of the app and one with this version and comparing them.

Needs review: @frosty
